cdk/lib/monitoring.ts

```diff
@@ -38,8 +38,8 @@ export class Monitoring extends GuStack {
 				functionName: `${lambdaBaseName}-${stage}`,
 				fileName: `${lambdaBaseName}-lambda-${region}.zip`,
 				handler: 'index.handler',
-				runtime: Runtime.NODEJS_14_X,
-				timeout: Duration.seconds(300),
+				runtime: Runtime.NODEJS_18_X,
+				timeout: Duration.seconds(500),
```

Comment:
Why was the timeout updated?

Reply:
Just for testing the lambda. Would expect to not increase them in the end